### PR TITLE
Add list of supported statements for Iceberg connector

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 1]
```diff
@@ -80,6 +80,52 @@ At a minimum, ``hive.metastore.uri`` must be configured:
...
+    * :doc:`/sql/select`
+    * :doc:`/sql/show-catalogs`
+    * :doc:`/sql/show-columns`
+    * :doc:`/sql/show-schemas`
+    * :doc:`/sql/show-tables`
+    * :doc:`/sql/describe`
+
+    Write access:
+
+    * :doc:`/sql/alter-schema`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/iceberg.rst:106]: I don't think this is supported.

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 2]
```diff
@@ -80,6 +80,52 @@ At a minimum, ``hive.metastore.uri`` must be configured:
...
+    * :doc:`/sql/alter-table`
+    * :doc:`/sql/create-schema`
+    * :doc:`/sql/create-table`
+    * :doc:`/sql/create-table-as`
+    * :doc:`/sql/delete`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/iceberg.rst:111]: Only partition level deletes are supported. No row-level arbitrary deletes.

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 3]
```diff
@@ -80,6 +80,52 @@ At a minimum, ``hive.metastore.uri`` must be configured:
...
+    * :doc:`/sql/drop-schema`
+    * :doc:`/sql/drop-table`
+    * :doc:`/sql/insert`
+    * :doc:`/sql/show-create-schema`
+    * :doc:`/sql/show-create-table`
+    * :doc:`/sql/update`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/iceberg.rst:117]: Not supported.

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 4]
```diff
@@ -80,6 +80,52 @@ At a minimum, ``hive.metastore.uri`` must be configured:
...
+
+    Other supported commands:
+
+    * :doc:`/sql/explain`
+    * :doc:`/sql/explain-analyze`
+    * :doc:`/sql/set-session`
+    * :doc:`/sql/show-functions`
+    * :doc:`/sql/show-session`
+    * :doc:`/sql/use`
+    * :doc:`/sql/values`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/iceberg.rst:127]: See https://github.com/trinodb/trino/pull/8419#discussion_r660919178

**Comment by Jessie212** [comment]
@hashhar This should be ready now.

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 5]
```diff
@@ -80,6 +80,61 @@ At a minimum, ``hive.metastore.uri`` must be configured:
...
+* :doc:`/sql/create-table`, see also :ref:`iceberg-tables`
+* :doc:`/sql/create-table-as`, see also :ref:`iceberg-tables`
+* :doc:`/sql/delete`, see also :ref:`iceberg-delete`
+* DROP MATERIALIZED VIEW
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-table`
+* REFRESH MATERIALIZED VIEW
+* SHOW CREATE MATERIALIZED VIEW
+
+Other supported commands:
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/iceberg.rst:121]: Use the recently added fragment for this section.

**Review by hashhar — APPROVED** [review]

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 6]
```diff
@@ -80,6 +80,48 @@ At a minimum, ``hive.metastore.uri`` must be configured:
     - Maximum number of partitions handled per writer.
     - 100
 
+SQL support
+-----------
+
+The connector provides **read access and limited write access** to data and
```
Comment (electrum) [inline docs/src/main/sphinx/connector/iceberg.rst:86]: We agreed that we'll follow up and move this into a common page

**Review by electrum — APPROVED** [review]

**Pull request merged**